### DISCUSSION TITLE
OAuth App Token Limits

### DISCUSSION THREAD

**Discussion by RMacfarlane** [community/community#24584] [Apps, API and Webhooks]
<p>From <a href="https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/#creating-multiple-tokens-for-oauth-apps" rel="nofollow noopener">https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/#creating-multiple-tokens-for-oauth-apps</a>:</p>
<p>&gt; There is a limit to the number of tokens that are issued per user/application/scope combination. If your application requests enough tokens to go over one of the limits, older tokens <em>with the same scope being requested</em> will stop working.</p>
<p>What exactly is this limit?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>HI <a class="mention" href="/rmacfarlane">@rmacfarlane</a>,</p>
<p>Thanks for being here! As far as I know the hard limit is 100.</p>
